src/components/look-controls.js

```diff
@@ -211,6 +232,17 @@ module.exports.Component = registerComponent('look-controls', {
   onMouseDown: function (event) {
     this.mouseDown = true;
     this.previousMouseEvent = event;
+
+    var sceneEl = this.el.sceneEl;
+    var canvasEl = sceneEl.canvas;
+
+    if (!this.pointerLocked) {
```

Comment:
see [this comment below](https://github.com/aframevr/aframe/pull/1056/files#r54680634)